manifest.md

```diff
@@ -92,7 +92,7 @@ A client will distinguish a manifest list from an image manifest based on the Co
 ## Example Manifest List
 
 *Example showing a simple manifest list pointing to image manifests for two platforms:*
-```json
+```application/vnd.oci.image.manifest.list.v1+json
```

Comment:
Looks like there is a way to preserve it:

```json,mediatype=asdf&foo=with%20space
{
  "id": "asdf"
}
```

This has ````json,mediatype=asdf&foo=with%20space` as the opening stanza. This will let us tag with other info, such as digests and titles to improve output.